.github/workflows/ci.yml

```
@@ -11,227 +11,153 @@ concurrency:
-            compiler: gcc-14
-          - runs-on: macos-15
-            compiler: gcc-15
-    uses: munich-quantum-toolkit/workflows/.github/workflows/reusable-cpp-tests-macos.yml@89734354f64f30a80dd16602d4e271df34348987 # v1.17.4
+        runs-on: [ubuntu-24.04, ubuntu-24.04-arm, macos-14]
+    uses: munich-quantum-toolkit/workflows/.github/workflows/reusable-python-packaging-wheel-cibuildwheel.yml@a86b62d384ceafdda21f1b7d49a702d6a312e58f # TODO: update hash and version here v1.17.x
     with:
       runs-on: ${{ matrix.runs-on }}
-      compiler: ${{ matrix.compiler }}
-      config: ${{ matrix.config }}
+      setup-mlir: true
+      llvm-version: ${{ env.LLVM_VERSION }}
```

actionlint (1.7.9)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)